src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlFileGenerator.cs

```diff
@@ -98,8 +98,9 @@ internal partial class XamlFileGenerator
         private readonly INamedTypeSymbol _setterSymbol;
 
         private readonly List<INamedTypeSymbol> _markupExtensionTypes;
+		private readonly List<INamedTypeSymbol> _xamlConversionTypes;
```

Comment:
why is the whitespace weird here? And below as well

Reply:
Fixed